database/migrations/default/1738010149899_crashes_list_comp_costs/up.sql

```diff
@@ -0,0 +1,245 @@
+drop view if exists locations_list_view;
+drop view if exists crashes_list_view;
+
+--
+-- add est_total_person_comp_cost to crashes_list_view
+--
+create view crashes_list_view as
+  WITH geocode_status AS (
+         SELECT cris.id,
+            unified.latitude IS NOT NULL AND unified.latitude IS DISTINCT FROM cris.latitude OR unified.longitude IS NOT NULL AND unified.longitude IS DISTINCT FROM cris.longitude AS is_manual_geocode
+           FROM crashes_cris cris
+             LEFT JOIN crashes unified ON cris.id = unified.id
+        )
+ SELECT crashes.id,
+    crashes.cris_crash_id,
+    crashes.record_locator,
+    crashes.case_id,
+    crashes.crash_timestamp,
+    crashes.address_primary,
+    crashes.address_secondary,
+    crashes.private_dr_fl,
+    crashes.in_austin_full_purpose,
+    crashes.location_id,
+    crashes.rpt_block_num,
+    crashes.rpt_street_pfx,
+    crashes.rpt_street_sfx,
+    crashes.rpt_street_name,
+    crashes.rpt_sec_block_num,
+    crashes.rpt_sec_street_pfx,
+    crashes.rpt_sec_street_sfx,
+    crashes.rpt_sec_street_name,
+    crashes.latitude,
+    crashes.longitude,
+    crashes.light_cond_id,
+    crashes.wthr_cond_id,
+    crashes.active_school_zone_fl,
+    crashes.schl_bus_fl,
+    crashes.at_intrsct_fl,
+    crashes.onsys_fl,
+    crashes.traffic_cntl_id,
+    crashes.road_constr_zone_fl,
+    crashes.rr_relat_fl,
+    crashes.toll_road_fl,
+    crashes.intrsct_relat_id,
+    crashes.obj_struck_id,
+    crashes.crash_speed_limit,
+    crashes.council_district,
+    crashes.is_temp_record,
+    crash_injury_metrics_view.nonincap_injry_count,
+    crash_injury_metrics_view.poss_injry_count,
+    crash_injury_metrics_view.sus_serious_injry_count,
+    crash_injury_metrics_view.non_injry_count,
+    crash_injury_metrics_view.tot_injry_count,
+    crash_injury_metrics_view.vz_fatality_count,
+    crash_injury_metrics_view.cris_fatality_count,
+    crash_injury_metrics_view.law_enf_fatality_count,
+    crash_injury_metrics_view.fatality_count,
+    crash_injury_metrics_view.unkn_injry_count,
+    crash_injury_metrics_view.est_comp_cost_crash_based,
+    crash_injury_metrics_view.est_total_person_comp_cost,
```

Comment:
here we `est_total_person_comp_cost`. `est_comp_cost_crash_based` was already available.